xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -1354,14 +1354,18 @@ HDR_STATUS DX::DeviceResources::ToggleHDR()
   DXGI_MODE_DESC md = {};
   GetDisplayMode(&md);
 
+  // Xbox does not like re-create swapchain but since full screen
+  // exclusive mode is not used, it's not necessary to do so.
+  const bool isXbox = (CSysInfo::GetWindowsDeviceFamily() == CSysInfo::Xbox);
+
```

Comment:
What does it mean "does not like"? same black screen + slowness you mentioned in my PR with ClearState()?

Reply:
No, in this case 4K resolution is lost on GUI. The video output is 4K but GUI is blur like 1080p upscaled to 4K.

The effect is very similar that when Windows limits memory usage or app not has 
<rescap:Capability Name="hevcPlayback" /> in the manifest and limits memory and graphics resources usage.

See: https://github.com/xbmc/xbmc/pull/19659#issuecomment-869806412

Reply:
Off topic here: probably for your PR one path is apply 10 bit setting without kill / recreate swap chain at least for Xbox. Probably current PR code changes are fine for x64.

Reply:
That's weird. So the screen resolution is 4k, the resolution of video played is 4k but GUI on top is like 1080p?
Could you write that explanation in the comment instead of "does not like".
What is the connection with FSE? Switching between FSE and windowed doesn't recreate the swap chain.

Is xbox always using 10 bit format for SDR? Otherwise the backbuffer needs to change from 8 to 10bit for HDR playback using m_swapchain->ResizeBuffers() but the current code makes it difficult to do that without a swap chain recreation.

off topic, for v22: even for x64 I think the swap chain is sometimes recreated needlessly, and m_swapchain->ResizeBuffer() would be enough most of the time, but risky to change now.

Reply:
> Is xbox always using 10 bit format for SDR

Yes, is the default when HDR display is detected: 10 bits is used for both HDR and SDR